src/openrct2/peep/Staff.cpp

```diff
@@ -653,9 +653,7 @@ bool Staff::DoHandymanPathFinding()
     }
 
     PeepDirection = newDirection;
-    DestinationX = chosenTile.x + 16;
-    DestinationY = chosenTile.y + 16;
-    DestinationTolerance = 3;
+    SetDestination(chosenTile + CoordsXY{ 16, 16 }, 3);
```

Comment:
Can we make these toTileCentre?

Reply:
It's not guaranteed to be the same, as NextLoc is a CoordsXY as well, IIRC.

Reply:
We should go through at some point and either enforce using only to tile centre or clearly mark why it's not correct in this situation. Guess we can leave for later though.